coalib/bearlib/abstractions/Linter.py

```diff
@@ -340,15 +378,17 @@ def process_output_corrected(self,
             Processes the executable's output as a corrected file.
 
             :param output:
-                The output of the program as a string.
+                The output of the program as a string containing the
+                corrected file corresponding to the file named ``filename``.
             :param filename:
                 The filename of the file currently being corrected.
             :param file:
                 The contents of the file currently being corrected.
             :param diff_severity:
-                The severity to use for generating results.
+                The severity to use for generating results, needs to be
+                defined inside ``coalib.results.RESULT_SEVERITY``.
             :param result_message:
-                The message to use for generating results.
+                The message-string to use for generating results.
```

Comment:
1. different commit as unrelated change
2. actually I'm not sure this change is improving docs :/